config-editor/config-editor-ui/src/app/components/testing/config-testing/config-testing.component.ts

```diff
@@ -24,13 +24,35 @@ export class ConfigTestingComponent implements OnInit {
   public isInvalid = false;
   public output: any;
 
+  private CONFIG_TESTER_KEY = "config_tester";
+  fields: FormlyFieldConfig[] = [ 
```

Comment:
maybe rename as well, there is already a 'field' variable, so could be confusing